kilkari/src/main/java/org/motechproject/nms/kilkari/domain/WeeklyCallsNotAnsweredMsisdnRecord.java

```diff
@@ -0,0 +1,51 @@
+package org.motechproject.nms.kilkari.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Unique;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+/**
+ * Record of Weekly Calls Not Answered Deactivated Numbers
+ */
+@Entity(tableName = "nms_weekly_calls_not_answered_msisdn")
+public class WeeklyCallsNotAnsweredMsisdnRecord {
+
+    @Field
+    @Unique
```

Comment:
Are you sure that you want this to be unique? You are basically trying to make this a primary key for this table.